### PR TITLE
Map 170 managing standards

### PR DESCRIPTION
**Pull request by ratheeshri1984**

### CODE REVIEW

File: src/Approvals/SFA.DAS.Approvals/Application/Providers/Queries/GetProvidersQueryHandler.cs [hunk 1]
```diff
@@ -11,19 +11,38 @@ namespace SFA.DAS.Approvals.Application.Providers.Queries
...
+            if (_featureToggles.RoatpStandardsEnabled)
+            {
+                var result = await _roatpApiClient.Get<GetRoatpProvidersListResponse>(new GetProvidersRequest());
+
+                return new GetProvidersResult
+                {
+                    Providers = result.RegisteredProviders
+                };
+            }
+            else
```
Comment (cofaulco) [inline src/Approvals/SFA.DAS.Approvals/Application/Providers/Queries/GetProvidersQueryHandler.cs:37]:
```
IEnumerable<GetProvidersListItem> providers;

            if (_featureToggles.RoatpStandardsEnabled)
            {
                var result = await _roatpApiClient.Get<GetRoatpProvidersListResponse>(new GetProvidersRequest());

                providers = result.RegisteredProviders;
            }
            else
            {
                var result = await _apiClient.Get<GetProvidersListResponse>(new GetProvidersRequest());

                providers = result.Providers;
                
            }

            return new GetProvidersResult
            {
                Providers = providers
            };
```

Comment (cofaulco) [reply src/Approvals/SFA.DAS.Approvals/Application/Providers/Queries/GetProvidersQueryHandler.cs:37]: could simplify this to a single return

File: src/Shared/SFA.DAS.SharedOuterApi/Configuration/FeatureToggles.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SFA.DAS.SharedOuterApi.Configuration
+{
+    public class FeatureToggles
+    {
+        public bool RoatpStandardsEnabled { get; set; }
```
Comment (cofaulco) [inline src/Shared/SFA.DAS.SharedOuterApi/Configuration/FeatureToggles.cs:9]: Why is the feature toggle referring to "standards" when we are getting "Providers". rename this and modify the config please?

**Review by cofaulco — CHANGES_REQUESTED** [review]

File: src/Approvals/SFA.DAS.Approvals/Application/Providers/Queries/GetProvidersQueryHandler.cs (hunk 1 above)
Comment (ratheeshri1984) [reply src/Approvals/SFA.DAS.Approvals/Application/Providers/Queries/GetProvidersQueryHandler.cs]: Done

File: src/Shared/SFA.DAS.SharedOuterApi/Configuration/FeatureToggles.cs (hunk 2 above)
Comment (ratheeshri1984) [reply src/Shared/SFA.DAS.SharedOuterApi/Configuration/FeatureToggles.cs]: Done

**Review by cofaulco — APPROVED** [review]

**Review by cofaulco — APPROVED** [review]

**Pull request merged**